spec/factories.rb

```diff
@@ -8,6 +8,10 @@
     password_confirmation 'abc123'
     date_of_birth Date.new(1970, 1, 1)
     gender 'Computer'
+
+    trait :admin do
+      roles { [FactoryGirl.create(:role, :admin)] }
```

Comment:
Split this up into two PRs and decided with the new enum style we are going to handle this differently so just dropped the entire commit.